.circleci/config.yml

```diff
@@ -9,21 +9,11 @@ jobs:
     steps:
       - checkout
 
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "Gemfile.lock" }}
```

Comment:
If you change this to either `v1-dependencies-{{ checksum "devise_2fa.gemspec" }}` it should work as expected again if you would like a faster build in the future.